addons/website_slides_survey/models/survey_survey.py

```diff
@@ -1,11 +1,47 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import models
+from odoo import models, fields, api
 
 
 class Survey(models.Model):
     _inherit = 'survey.survey'
+    slide_ids = fields.One2many('slide.slide', 'survey_id', 'Certification')
```

Comment:
As there can be many, I would change to "Certifications"

Reply:
Done